src/components/GridTemplates/DetailsPage1/DetailsPage1.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @file Details page template #1.
+ * @copyright IBM Security 2020
+ */
+
+import classnames from 'classnames';
+import { string, node } from 'prop-types';
+import React from 'react';
+
+import { Column, Grid, Row } from 'carbon-components-react/lib/components/Grid';
+
+import { getComponentNamespace } from '../../../globals/namespace';
+
+const namespace = getComponentNamespace('details-page-1');
+
+function DetailsPage1Main({ children, className, ...other }) {
+  const classes = classnames(`${namespace}__main`, className);
+  return (
+    <Column lg={16} className={classes} {...other}>
+      {children}
+    </Column>
+  );
+}
+
+DetailsPage1Main.propTypes = {
+  children: undefined,
+  className: undefined,
+};
+
+DetailsPage1Main.defaultProps = {
+  children: node,
+  className: string,
+};
+
+function DetailsPage1Section({ children, className, ...other }) {
+  const classes = classnames(`${namespace}__section`, className);
+  return (
+    <Column sm={4} md={8} lg={6} className={classes} {...other}>
+      {children}
+    </Column>
+  );
+}
+
+DetailsPage1Section.propTypes = {
+  children: undefined,
+  className: undefined,
+};
+
+DetailsPage1Section.defaultProps = {
+  children: node,
+  className: string,
+};
+
+function DetailsPage1SectionTerm({ children, className, ...other }) {
```

Comment:
This name is... so much 😆 
I was thinking of "term" and "definition" for this plus `DetailsPage1SectionDefinition`, since they are side-by-side nested columns. But again I am VERY open to changing these names.

Reply:
Good point! This is definitely dependent on the kind of content @cameroncalder expects will most likely be contained in these columns